Dockerfile

```diff
@@ -19,7 +19,7 @@ WORKDIR $GHOST_INSTALL
 
 # We use SQLite as our DB. Force install "sqlite3" manually since it's an optional dependency of "ghost"
 RUN set -eux                                                    && \
-    apk update && apk add su-exec                               && \
+    apk update && apk add su-exec python make                   && \
```

Comment:
Here you are the installation error:
```
gyp WARN EACCES attempting to reinstall using temporary dev dir "/usr/local/lib/node_modules/ghost-cli/node_modules/dtrace-provider/.node-gyp"
gyp WARN EACCES user "nobody" does not have permission to access the dev dir "/usr/local/lib/node_modules/ghost-cli/node_modules/dtrace-provider/.node-gyp/8.9.4
```
It is in infinite loop on this error (you can read it just changing the error log level):
```
npm install --loglevel=info -g ghost-cli
```

Reply:
Thanks, I'll look into this, but it compiles fine on my end, I don't know why it doesn't. Probably because I used the base ``alpine:latest`` @mmornati